libs/permission-checker/src/Check/RunnerSyncApi/CanRunAction.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\PermissionChecker\Check\RunnerSyncApi;
+
+use Keboola\PermissionChecker\BranchType;
+use Keboola\PermissionChecker\Exception\PermissionDeniedException;
+use Keboola\PermissionChecker\Feature;
+use Keboola\PermissionChecker\PermissionCheckInterface;
+use Keboola\PermissionChecker\Role;
+use Keboola\PermissionChecker\StorageApiToken;
+
+class CanRunAction implements PermissionCheckInterface
+{
+    public function __construct(
+        private readonly BranchType $branchType,
+        private readonly string $componentId,
+    ) {
+    }
+
+    public function checkPermissions(StorageApiToken $token): void
+    {
+        if (!$token->hasFeature(Feature::QUEUE_V2)) {
+            throw PermissionDeniedException::missingFeature(Feature::QUEUE_V2);
+        }
```

Comment:
Tohle pravidlo nemuzu nikde najit, podle ceho to je? Jako vypada to logicky. Na druhou stranu technicky k tomu nijak `queuev2` neni potreba (to nebezi na queue) a zadne bussiness omezeni k otmu taky neni, ne?

Reply:
To tam sice nebylo, ale vzal jsem to odsud https://github.com/keboola/platform-libraries/blob/90684322e461f785d769d4a072e89ba8ca0a55f2/libs/permission-checker/src/Check/JobQueue/CanRunJob.php#L24 z `CanRunJob` jsem vycházel a přijde mi že by to mělo být stejné

Reply:
Jo, u toho `CanRunJob` to je normalne vytazeny z queue a je to tam proto, ze bez te feature proste nepustis job na queuev2. Ale sync akci klidne pustit muzes (technicky). Nevim jestli existuje nejaky ude-case, ze by se mohla/nesmela poustet sync akce bez `queuev2` feature. Jen me to trklo, protoze jsem to nikde nenasel (= je to nove)

Reply:
co si myslí  @odinuv

Reply:
No ten check je tam spíš kvůli konzistenci, reálně by ničemu nevadilo si spustit v QV1 projektu job na QV2, fungovalo by to. Akorát bys měl v projektu bordel. Sync akce nic nikam nezapisují a tak se můžou pouštět tak i tak, dokonce si matně vzpomínám, že se to k něčemu párkrát použilo (nějakej debug komponenty s tagem, kterej nejde na QV1 spustit).

Reply:
No tak ja to odbereu no. Ale pro zaznam sem uvadim ze nesouhalsim s tim aby to bylo nekonzistentni a melo by se to chovat stejne jako pousteni jobu :D

Reply:
Objection noted :)